src/getDataFromURL.js

```diff
@@ -0,0 +1,6 @@
+export async function getDataFromURL(url) {
+  const response = await fetch(url)
+  const data = await response.json()
+
+  return data
```

Comment:
I think you can do `return await response.json()`

Reply:
Avoid using the `data` word, everything is data or info. This could be `fetchUrl` or something like that.